cni/network/invoker_cns.go

```diff
@@ -118,6 +122,8 @@ func (invoker *CNSIPAMInvoker) Add(addConfig IPAMAddConfig) (IPAMAddResult, erro
 	}
 
 	addResult := IPAMAddResult{}
+	// Default address type will be the default interface unless isDefaultInterface is true for a secondary address
```

Comment:
This is a confusing comment. Ideally we should not be using isDefaultInterface to determine the routes